### ISSUE TITLE
Overides eslintrc with incorrect changes

### ISSUE THREAD

**Issue by wopian** [codeclimate/codeclimate-eslint#238]
Continued from https://github.com/codeclimate/codeclimate-eslint/issues/234#issuecomment-290227745.

Recently re-enabled the ESLint engine as it finally supported the `html` plugin needed to parse `<script>` within `.vue` files. 

However ESLint produces several dozen issues about failing to parse various things (mostly within `<template>` which should be being ignored. E.g:

![](http://i.imgur.com/82Qgpnz.png)

Viewing a build within the [recent builds](https://codeclimate.com/github/wopian/hibari/builds) page on codeclimate revealed that my ESLint config is getting modified with the following changes:

![](http://i.imgur.com/mwslby4.png)

```diff
-root: true
-parser: babel-eslint
 parserOptions:
   sourceType: module
   ecmaFeatures:
-    jsx: false
-extends: standard
-plugins:
-- html
+    jsx: true
```

While `jsx` being changed from `false` to `true` isn't a breaking change, the removal of `plugins: - html` is and causes the entire `.vue` file to be parsed as JS. It's possible this is also the cause of `jsx` getting re-enabled too.

Removal of `extends: standard` is OK, as its just adding all the rules to `.eslintrc.yml`.

[`.eslintrc.yml`](https://github.com/wopian/hibari/blob/master/.eslintrc.yml):

```yml
root: true
parser: babel-eslint
parserOptions:
  sourceType: module
  ecmaFeatures:
    jsx: false
extends: standard
plugins:
- html
rules:
  # Allow paren-less arrow functions
  arrow-parens: 0
  # Allow async-await
  generator-star-spacing: 0
```

[`client/.eslintrc.yml`](https://github.com/wopian/hibari/blob/master/client/.eslintrc.yml):

```yml
env:
  browser: true
globals:
  VERSION: true
```

[`test/unit/.eslintrc.yml`](https://github.com/wopian/hibari/blob/master/test/unit/.eslintrc.yml):

```yml
env:
  mocha: true

globals:
  expect: true
  sinon: true
```

Snippet of [`.codeclimate.yml`](https://github.com/wopian/hibari/blob/master/.codeclimate.yml):

```yml
engines:
  eslint:
    enabled: true
    config:
      # config: .eslintrc.yml
      extensions:
      - .js
      - .vue
ratings:
  paths:
  - "**.js"
  - "**.vue"
```

**Comment by dblandin** [comment]
Hey @wopian,

Thank you for the issue! This is actually an issue with the CLI itself as inferred configuration are implemented there. I've opened up a PR on that repo which is waiting on review: https://github.com/codeclimate/codeclimate/pull/610

I'll ping back here when we've released a new version including the fix. Thanks!

**Comment by dblandin** [comment]
Hey @wopian,

I just released v0.59.0 of the CLI which includes a fix for this issue: https://github.com/codeclimate/codeclimate/pull/613

Thanks for the report!
